assets/airports/kpdx.json

```diff
@@ -91,6 +91,189 @@
                 ["N45d08m43.863", "W122d14m10.743"],
                 ["N45d07m16.326", "W122d18m11.921"]
             ]
+        },
+        { // SE section
+            "floor": 17,
+            "ceiling": 40,
+            "airspace_class": "C",
+            "poly": [
+                ["N45.51233333", "W122.63383333"], // LU
```

Comment:
I'm using U,D,L,R to denote up, down, left, right. This way I can keep track of the shared points between the shelves.